ios/FluentUI/Avatar/AvatarGroupView.swift

```diff
@@ -41,6 +41,15 @@ open class AvatarGroupView: UIView {
         }
     }
 
+    /// Set to true to generate border colors from InitialView ColorSet
```

Comment:
except for the overflow avatarView case right?

Reply:
Yes.. that's intended right?

Reply:
yeah make sure you clarify that in the documentation